static/flow/views/pi-selector-panel.js

```diff
@@ -678,9 +678,9 @@ var PiSelectorPanel = function(options) {
 
         console.log("[DEBUG] Stopping program", _this.currentRecordingLocation);
```

Comment:
I think this should also be changed to `console.log("[DEBUG] Stopping program", recordingLocation);` otherwise it might cause confusion in later debugging.

Reply:
Good catch.  Looking at this again makes me think there might be one more workflow path that doesn't work.  I'm going to test a little more before adding the next commit.